exporter/loadbalancingexporter/consistent_hashing_test.go

```diff
@@ -48,7 +48,8 @@ func TestEndpointFor(t *testing.T) {
 	} {
 		t.Run(fmt.Sprintf("Endpoint for traceID %s", tt.traceID.HexString()), func(t *testing.T) {
```

Comment:
Can you also add a new case for services, just to have an explicit test with that data format? It shouldn't matter, as it's all bytes, but explicit test documents the supported use cases.